run.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+include 'vendor/autoload.php';
+include 'funcs_scripts.php';
+include 'funcs_utils.php';
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+// consts
+const CURRENT_CMS_MAJOR = '5';
+const BRANCH_OPTIONS = ['next-minor', 'next-patch'];
+const DEFAULT_BRANCH = 'next-minor';
+const DEFAULT_ACCOUNT = 'creative-commoners';
+const DATA_DIR = '_data';
+const MODULES_DIR = '_modules';
+const TOOL_URL = 'https://github.com/silverstripe/module-standardiser';
+const PR_TITLE = 'MNT Run module-standardiser';
+const PR_DESCRIPTION = 'This pull-request was created automatically by [module-standardiser](' . TOOL_URL . ')';
+
+// global variables
+$MODULE_DIR = '';
+$PRS_CREATED = [];
+$REPOS_WITH_PRS_CREATED = [];
+$OUT = null;
+
+$app = new Application();
+$app->register('update')
+    ->setDescription('The main script of module-standardiser')
+    ->addOption(
+        'cms-major',
+        null,
```

Comment:
I intentionally left out the shortcut options because they're less obvious what you're doing e.g. instructions that say `MS_GITHUB_TOKEN=abc123 php run.php update --c=5 --b=next-minor --d --o=silverstripe-config,silverstripe-assets` is siginficantly less obviously what's going on